.github/workflows/container-healthchecks.yml

```
@@ -0,0 +1,37 @@
+name: "PR: Run Container Health Checks"
+
+on:
+  # Allow manual triggering
+  workflow_dispatch:
```

This action is triggered only manually. I suggest running calling it from `.github/workflows/continuous-integration.yml`, in which case it can be renamed:
```suggestion
name: "Container Health Checks"

on:
  # Allow manual triggering
  workflow_dispatch:
```